scalalib/src/publish/VersionScheme.scala

```diff
@@ -0,0 +1,11 @@
+package mill.scalalib.publish
+
+sealed abstract class VersionScheme(val value: String)
```

Comment:
What do you think of a `def toProperty: (String, String) = "info.versionScheme -> value`? This would make the encoding very explicit.